### DISCUSSION TITLE
The OG plots of harvester can not show the numbers in GUI

### DISCUSSION THREAD

**Discussion by tracy32792** [Chia-Network/chia-blockchain#7209] [Q&A]
The OG plots of harvester can not show the numbers in GUI.what can i do something to solve the problem？

**Comment by ZielReaver** [comment]
is it that the plots themselves do not show up or some are missing. I know I had the same problem and had to wait about 10mins then under plots tab at the config button in top right I used the refresh plots option and they showed up. This had freaked me out because I updated then was not seeing the 1700sh plots I had in GUI anymore for my OG stuff.
